### PR TITLE
Added a new caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @cody-carlson
- [ ] @cody-carlson-learning
- [ ] @BRGravley
- [ ] @BrightTigger
- [ ] @CIMSims
- [ ] @DEPearson
- [ ] @DavidH102
- [ ] @GaerynDaenyr
- [ ] @bhaveshshah21-home
- [ ] @bnshawtest
- [ ] @brangull51
- [ ] @cfranchi
- [ ] @dually8
- [ ] @nunezros
- [ ] @pmconnolly80
- [ ] @rbmiller12
- [ ] @thetechlink
- [ ] @vanand11

### PR DESCRIPTION
**Pull request by ghost**
TL;DR This PR is updating a copy of the 11-bnshawtest.md file.  

This PR resolves / closes / fixes #2 

## Done
- made a commit
- created a branch
- sample item here

## To Do
[ ] merge the branch
[ ] delete the branch after the merge

### CODE REVIEW

File: _slides/11-bnshawtest.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+A new caption
```
Comment (DEPearson) [inline _slides/11-bnshawtest.md:6]: Looks Good

**Review by DEPearson — APPROVED** [review]
Looks great

**Review by rbmiller12 — APPROVED** [review]
Sweet code! 💯

**Comment by rbmiller12** [comment]
Great to finally see this change made!

**Pull request merged**